### PR TITLE
Feature/577 landing page sections

### PR DESCRIPTION
**Pull request by prusovak2**
related issue https://github.com/cityvizor/cityvizor/issues/577

### CODE REVIEW

File: client/src/app/views/admin/views/admin-profile/admin-profile-settings/admin-profile-settings.component.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <div class="row">
   <div class="col-md-7">
     <h3>Základní informace</h3>
+    <div *ngIf="(profile.status === 'visible' || profile.status === 'pending') && (profile.parent == null && profile.sectionId == null)" class="pb-3">
+      <span class="label label-red">Tento profil je {{profile.status == 'visible' ? 'viditelný' : 'připravovaný'}},  ale nemá nastavený rodičovský profil ani sekci na hlavní stránce, <br> ve které by měl být zobrazen. Uživatelé aplikace tento profil aktuálně nevidí.</span>
```
Comment (oroztocil) [inline client/src/app/views/admin/views/admin-profile/admin-profile-settings/admin-profile-settings.component.html:5]: 👍 👍 👍

File: server/src/routers/admin/sections.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+);
+
+// CREATE
+router.post('/', acl('options:write'), async (req: Request, res: Response) => {
+    try {
+        const body: SectionRecord = req.body;
+        if(!body){
+            res.status(400).json({error: "Invalid 'selection' request."});
+        }
+        console.log("$$$$$$$$$$$$$$ HERE")
```
Comment (oroztocil) [inline server/src/routers/admin/sections.ts:49]: Hello there

File: server/src/routers/public/profiles.ts [hunk 3]
```diff
@@ -33,35 +34,58 @@ function createQueryWithStatusFilter(statuses, tableName: string) {
   return query;
 }
 
+function addCountChildrenInnerQuery(query, statusses) {
+  const innerQuery = createQueryWithStatusFilter(
+    statusses,
```
Comment (oroztocil) [inline server/src/routers/public/profiles.ts:39]: statuses

**Review by oroztocil — APPROVED** [review]
Super! Odstran si ty printy v `server/src/routers/admin/sections.ts` a 🚀

File: server/src/routers/admin/sections.ts (hunk 2 above)
Comment (prusovak2) [reply server/src/routers/admin/sections.ts:49]: deleted

File: server/src/routers/public/profiles.ts (hunk 3 above)
Comment (prusovak2) [reply server/src/routers/public/profiles.ts:39]: 🐍

**Review by oroztocil — APPROVED** [review]

**Pull request merged**